api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -2644,11 +2644,23 @@ service AuthService {
   rpc CreateResetPasswordToken(CreateResetPasswordTokenRequest) returns (types.UserTokenV3);
 
   // CreateBot creates a new bot user.
-  rpc CreateBot(CreateBotRequest) returns (CreateBotResponse);
+  //
+  // Deprecated: Use [teleport.machineid.v1.BotService] instead.
```

Comment:
How can I ensure that a doc link like this is valid?

This comment ends up in `api/client/proto/authservice.pb.go`. I was under the impression that if `machineid.v1` is not directly imported in that file, then a doc link must use the full import path, which I assume would be `github.com/gravitational/teleport/…`. Isaiah pointed it out in one of my PRs but I've never tried to actually verify if my doc links are correct.

https://tip.golang.org/doc/comment#doclinks

I know it doesn't matter much as most of the time those "doc links" will not end up in any docs and are just for internal reference. Still, it's been bugging me since it was pointed out to me so I figured maybe you know the answer.

Reply:
Ah - I suppose it's a little odd here since my intention was to "comment on the proto" and therefore here I've referred to the proto service rather than the generated code.